bindings/ruby/ext/dependencies.rb

```diff
@@ -17,7 +17,12 @@ def initialize(cmake, options)
   def libs
     tsort.filter_map {|node|
       label, shape = @nodes[node]
-      shape == @static_lib_shape ? label : nil
+      if shape == @static_lib_shape
+        clean_label = label.gsub(/\\n\([^)]+\)/, '').gsub(/[^a-zA-Z0-9_-]/, '-')
+        clean_label
```

Comment:
```suggestion
        label.gsub(/\\n\([^)]+\)/, '')
```

* `.gsub(/[^a-zA-Z0-9_-]/, '-')` seems not needed
* variable assignment to `clean_label` seems not needed

Reply:
@KitaitiMakoto Thanks!